main.go

```diff
@@ -30,7 +30,7 @@ import (
 	"path"
 	"strings"
 
-	"./protos"
+	"github.com/GoogleCloudPlatform/protoc-gen-bq-schema/protos"
```

Comment:
this is the only change made by my previous PR, which this supersedes, if the functionality is desired.